src/libs/ajax.js

```diff
@@ -471,43 +480,34 @@ const Billing = signal => ({
     return res
   },
 
-  billingProject: async projectName => {
-    const route = `billing/v2/${projectName}`
-    const res = await fetchRawls(route, _.merge(authOpts(), { signal, method: 'GET' }))
+  listProjectUsers: async projectName => {
+    const res = await fetchRawls(`billing/v2/${projectName}/members`, _.merge(authOpts(), { signal }))
     return res.json()
   },
 
-  project: projectName => {
```

Comment:
Why are we getting rid of the project wrapper here? That seemed more representative of the codebase's general practice. (Similar to analysis and notebooks under storage)

Reply:
I'm with @s-rubenstein on this one, I find the consistency with other places to be extremely valuable. It also cuts down on the number of positional arguments to functions.

Reply:
I disagree strongly. This way keeps things simple and follows the underlying API whereas the other is consistency for consistency's sake with something that honestly feels clunky.

Reply:
The benefit that it provides, in my mind, is similar to object layering. By abstracting out `projectUsers` (and I like that name a lot), we gain the ability to have a structure that groups more in a way that makes it easy to find what neighboring functions are, and keeps associated endpoints grouped. (If I see a `.projectUsers(project).add()` call, I can fairly easily follow `.projectUsers` to find all associated endpoints.

I would personally prefer those child objects pulled out into their own objects, so we get a cleaner view in our editors, which I think would more easily show the benefits here.

I think it does follow the underlying REST structure a bit more (where they are all a page) but it doesn't necessarily follow the route definitions in the service as cleanly. In Rawls (or Sam) these are grouped effectively by arg, so the original code here preserves that, IMO.